File: seccion_5/ej.py
class Contador:
    total = 0
    
    def __init__(self):
        Contador.total += 1
    
    @staticmethod
    def mostrar_total():
        print("Total de objetos:", Contador.total)
c1 = Contador()
c2 = Contador()
c3 = Contador()
c4 = Contador()
c5 = Contador()
Contador.mostrar_total()

File: seccion_5/ej.java
class Contador {
 static int total = 0;
 Contador() {
 total++;
 }
 static void mostrarTotal() {
 System.out.println("Total de objetos: " + total);
 }
}
public class ej {
 public static void main(String[] args) {
 new Contador();
 new Contador();
 new Contador();
 new Contador(); 
 new Contador();
 Contador.mostrarTotal();
 }
}